packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -536,3 +532,13 @@ function registerInteractionListener(
     addEventListener('click', registerInteractionTransaction, { once: false, capture: true });
   }
 }
+
+// We store the active idle span on the client object, so we can access it from exported functions
+const ACTIVE_IDLE_SPAN_PROPERTY = '_sentry_idleSpan';
+function getActiveIdleSpan(client: Client): Span | undefined {
+  return (client as { [ACTIVE_IDLE_SPAN_PROPERTY]?: Span })[ACTIVE_IDLE_SPAN_PROPERTY];
+}
+
+function setActiveIdleSpan(client: Client, span: Span): void {
```

Comment:
m: Do we need to add a possibility to unset the span on the client? On the scope this should be taken care of today but I'm afraid the two things could diverge (see my other comment).

Reply:
since we only do this internally here, we can keep/modify this API as needed - this is not exposed!